subscribie/utils.py

```diff
@@ -295,11 +297,7 @@ def stripe_invoice_failed_all_automated_collection_attempts(stripeInvoice):
     """Returns true/false if a Stripe Invoice has failed all collection attemts
     and no further *automated* collection will take place."""
     if stripeInvoice.subscribie_subscription_id:
-        if (
```

Comment:
The removal of those lines changes the meaning and purpose of the function `stripe_invoice_failed_all_automated_collection_attempts` which is to: Return true/false if a Stripe Invoice has failed all collection attemts  and no further *automated* collection will take place.
It's no longer returning invoices which have failed all automated attempts, instead it's now returning any open invoice.

I was expecting something more like this: (an additional function) 
Please see issue details : https://github.com/Subscribie/subscribie/issues/849#issuecomment-1087862098